nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/storage/ListGCSBucket.java

```diff
@@ -396,11 +401,31 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             listByTrackingTimestamps(context, session);
         } else if (BY_ENTITIES.equals(listingStrategy)) {
             listByTrackingEntities(context, session);
+        } else if (NO_TRACKING.equals(listingStrategy)) {
+            listNoTracking(context, session);
         } else {
             throw new ProcessException("Unknown listing strategy: " + listingStrategy);
         }
     }
 
+    private void listNoTracking(ProcessContext context, ProcessSession session) {
+        final long startNanos = System.nanoTime();
+        final ListingAction listingAction = new NoTrackingListingAction(context, session);
+
+        try {
+            listBucket(context, listingAction);
+        } catch (final Exception e) {
+            getLogger().error("Failed to list contents of GCS Bucket due to {}", new Object[] {e}, e);
```

Comment:
The `due to {}` syntax for exceptions should not be used, since the message will be incorporated in bulletin summaries and printed in stack traces.
```suggestion
            getLogger().error("Failed to list contents of GCS Bucket", e);
```

Reply:
Yeah, I followed the same approach as in other places in the code, will fix